apps/meteor/app/cloud/server/functions/removeLicense.ts

```diff
@@ -20,7 +20,8 @@ export async function removeLicense() {
 			throw new CloudWorkspaceAccessTokenEmptyError();
 		}
 
-		const workspaceRegistrationClientUri = settings.get<string>('Cloud_Workspace_Registration_Client_Uri');
+		const workspaceRegistrationClientUri = Settings.getValueById('Cloud_Workspace_Registration_Client_Uri');
```

Comment:
I think I'm missing an await here, alto some test to make sure we have this setting set

Reply:
are there any tests on these cloud functions?